net-fs/autofs/autofs-5.1.9-r1.ebuild

```diff
@@ -77,7 +76,6 @@ src_configure() {
 	local myeconfargs=(
 		--with-confdir=/etc/conf.d
 		--with-mapdir=/etc/autofs
-		$(use_with dmalloc)
```

Comment:
It's better to explicitly pass --without-dmalloc etc to avoid automagic even if it's no longer in-tree.

Reply:
configure doesn't check for --without-dmalloc
Thankfully, autotools had shred of sanity and don't implicitly enable it if present

Reply:
But explicitly disabled now